patch-package
--- a/src/generic/generic-component-file/generic-component-file.twig
+++ b/src/generic/generic-component-file/generic-component-file.twig
@@ -201,7 +201,7 @@
           {% if _href is empty %}
             {% set _button_extra_attributes = [{ 'name': 'type', 'value': 'button' }] %}
           {% else %}
-            {% set _button_extra_attributes = [{ 'name': 'download', 'value': 'file' }] %}
+            {% set _button_extra_attributes = [{ 'name': 'download', 'value': title }] %}
           {% endif %}
 
           {% include '@ecl/' ~ system ~ '-component-button' with {
